docs/integration-tutorial.md

```diff
@@ -0,0 +1,212 @@
+
+For this we have written the script [scripts/synthetix.js](scripts/synthetix.js) which essentaially performs the following sequence of operations:
+
+- Instantiate a synthetix and batchExchange for the desired network
+- Fetch relevant token infomation for `sUSD` and `sETH`
+- Fetch the exchange rate `sUSD`<->`sETH` from their on chain oracle along with the network fees for trading these tokens
+- Compare the buy and sell `sETH` for `sUSD` prices with thier counter parts on Gnosis Protocol
+- If the price comparisons suggest there is likely trade on and the "bot" has sufficient balance, place order(s) on BatchExchange valid for a single batch.
+
+All of these facts are documented in the script's logs. To test this, run
+
+```sh
+npx truffle exec scripts/synthetix.js --network rinkeby
+```
+
+Given that there is often no orders between these tokens on rinkeby, you should see the following logs:
+
+```
+Using network 'rinkeby'.
+
+Using account 0x627306090abaB3A6e1400e9345bC60c78a8BEf57
+Oracle sETH Price (in sUSD) 222.04638321287640657
+Gnosis Protocol sell sETH price (in sUSD) Infinity
+Gnosis Protocol buy  sETH price (in sUSD) 0
+Not placing buy  sETH order, our rate of 221.38024406323777 is too low  for exchange.
+Not placing sell sETH order, our rate of 222.15740640448283 is too high for exchange.
+```
+
+Now that we have this bot-script ready for production it remains run this automatically in every batch.
+For this we will publish this project as a docker image and run the script every five minutes as a cronjob on kubernetes.
+
+### Build a Docker Image
+
+The docker file is a very simple basic instance of this project having a bash entry point. [Dockerfile](Dockerfile).
+To build the image, from within the project root
+
+```sh
+docker build -t <YOUR_DOCKERHUB_HANDLE>/synthetix-bot .
+docker run -e INFURA_KEY=$YOUR_INFURA_KEY -e PK=$YOUR_PRIVATE_KEY -t <YOUR_DOCKERHUB_HANDLE>/synthetix-bot:latest "truffle exec scripts/synthetix.js --network rinkeby"
+```
+
+To avoid including `INFURA_KEY` on every execution, this value can be included/replaced line 16 of [truffle-config.js](truffle-config.js) before building the docker image.
+However, it is important to note that these keys should not be pushed into a public repo.
+
+### Running the Bot
+
+At this point, you are now ready to deploy your liquidity bot. This is easily done by running our script in a crontab every five minutes at, say, the 3 minute mark of each batch.
+Although we will not provide any instructions for deployment here, there is a sample [crontab.txt](contab.txt) and slightly altered [Dockerfile](Dockerfile.Cron) that can be used to run the job inside a container.
+
+Alternatively, see the kubernetes directory here for a mock deployment configuration.
```

Comment:
Since, we are not in the actual repo, the "here" makes not to much sense. I would just leave it out.

Reply:
I assume you are referring to this entire segment for building the image. Makes sense. Will remove!

Reply:
Done!